### DISCUSSION TITLE
Welcome to guess-my-word Discussions!

### DISCUSSION THREAD

**Discussion by hryanjones** [hryanjones/guess-my-word#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by haberflave** [comment]
Happy to find this. I remember the original version and used to play with my Brother. I took a stab at recreating it and talk myself python to do so. Ended up pretty limited with option to get the py script to do much else but run in a terminal. I gave a couple raspberry pis out to some friends, who love to play.
